render.c

```diff
@@ -131,7 +147,16 @@ static int render_notification(cairo_t *cairo, struct mako_state *state,
 	}
 	int text_height = buffer_text_height / scale;
 
-	int notif_height = border_size + padding_height + text_height;
+	int notif_height;
+#ifdef SHOW_ICONS
```

Comment:
Does this really need a guard? If icons are compiled out, the image would just always be NULL.

Reply:
As it's written now when icons are disabled every mention of them disappear.